config.js

```diff
@@ -90,14 +90,14 @@ module.exports = {
     // projects.
     ignoreContextHeader: false,
 
+    // The contents of a key file. If this field is set, its contents will be
+    // used for authentication instead of your application default credentials.
+    credentials: null,
+
```

Comment:
The authentication library used in `cloud-common` checks `credentials` and `keyFilename` in reverse order to ours.